agenthub/micro/math_agent/agent.yaml

```diff
@@ -0,0 +1,24 @@
+name: MathAgent
+description: "Solves simple and complex math problems using python"
+generates: Action
+container: python:3.12.3-bookworm
+inputs:
+  task: string
+outputs:
+  answer: string
+examples:
+  - input:
+      task: "What is 2 + 2?"
+    output:
+      answer: "4"
```

Comment:
Shouldn't you include code in the examples, instead of input-output pairs?

Reply:
It's a good question.

I'm not 100% sure what I want to do with these examples yet. Possibilities include:
* Documentation
* Filling out the prompt templates for regression purposes
* Testing

But it's probably best to focus solely on inputs/outputs. E.g. the script could just as easily be:
```python
a = 2
b = 2
print(a + b)
```

Reply:
I feel like we should include the examples in the prompt?